numpy/testing/utils.py

```diff
@@ -44,6 +44,28 @@ class KnownFailureException(Exception):
 verbose = 0
 
 
+def import_nose():
+    """ Import nose only when needed.
+    """
+    fine_nose = True
```

Comment:
Ah, wonked up the order a bit. A later commit removes the duplicate function so that only this one remains (i.e. I moved it in reality).